server/src/controllers/job.controller.ts

```diff
@@ -21,10 +23,10 @@ export class JobController {
   @Endpoint({
     summary: 'Retrieve queue counts and status',
     description: 'Retrieve the counts of the current queue, as well as the current status.',
-    history: new HistoryBuilder().added('v1').beta('v1').stable('v2'),
+    history: new HistoryBuilder().added('v1').beta('v1').stable('v2').deprecated('v2.3.0'),
```

Comment:
I guess this should also be 2.4.0? Introducing new endpoints and in the same version deprecating the old ones